### DISCUSSION TITLE
No specs found for module "lazyvim.plugins.extras.coding.codeium"

### DISCUSSION THREAD

**Discussion by gerroon** [LazyVim/LazyVim#5000] [Q&A] [answered]
Hi

I thought I disabled this but it keeps showing me this message during startup, I am on windows. I am wondering if anyone can help me out with getting tid of this message.

```
Error detected while processing ......\nvim\.init.lua
No specs found for module "lazyvim.plugins.extras.coding.codeium"


```

init.lua
```
-- bootstrap lazy.nvim, LazyVim and your plugins
require("config.lazy")

```

lua\plugins\disabled.lua
```
return {
  -- disable trouble
  { "Exafunction/codeium.nvim", enabled = false },
  
}
```

**Comment by folke** [comment, marked as answer]
`lazyvim.plugins.extras.coding.codeium` -> `lazyvim.plugins.extras.ai.codeium`.
You either have an import of this in your `lazy.lua` or `lazyvim.json`

**Reply by gerroon** [reply]
Thank you, I see that the line existsi in lazyvim.json and I have no idea why it is in there given I never edited these files myself.

 ```
{
  "extras": [
    "lazyvim.plugins.extras.coding.codeium",
    "lazyvim.plugins.extras.editor.aerial"
  ],
  "news": {
    "NEWS.md": "7637"
  },
  "version": 7
}

```
